html/static/script.js

```diff
@@ -161,3 +161,14 @@ function updateSample(element, uid) {
       alert('Error in updating the dictionary: ' + data.responseText);
     })
 }
+
+function changePage(element, page) {
+  var loc = location.href;
```

Comment:
You should use `let` for variables in JavaScript. Some historical reason that `var` is bad and `let` is good.